utils/call-gen/src/lib.rs

```diff
@@ -44,8 +45,12 @@ pub enum GearCall {
     UploadCode(UploadCodeArgs),
 }
 
-pub fn generate_gear_program<Rng: CallGenRng>(seed: u64) -> Vec<u8> {
+pub fn generate_gear_program<Rng: CallGenRng>(
+    existing_programs: Vec<ProgramId>,
```

Comment:
due to `NonEmpty` type usage in the crate to state existing programs/codes, I'd state in the comment, that call to function accepts empty program vector too